static/js/subs.js

```diff
@@ -275,7 +279,7 @@ function add_sub_to_table(sub) {
     add_email_hint(sub, email_address_hint_content);
 
     if (meta.stream_created) {
-        $(".stream-row[data-stream-name='" + meta.stream_created + "']").click();
+        $(".stream-row[data-stream-id='" + meta.stream_id + "']").click();
```

Comment:
`meta.stream_id` isn't set.  I think you need this to do something more complicated, like `get_sub(meta.stream_created).stream_id`

(the whole point is that when we submit the request to create the stream, we don't have a stream ID yet since that new stream hasn't been assigned an ID).